addons/account/models/account_tax.py

```diff
@@ -124,8 +124,20 @@ class AccountTax(models.Model):
     amount = fields.Float(required=True, digits=(16, 4), default=0.0, tracking=True)
     description = fields.Html(string='Description', translate=html_translate)
     invoice_label = fields.Char(string='Label on Invoices', translate=True)
-    price_include = fields.Boolean(string='Included in Price', default=False, tracking=True,
-        help="Check this if the price you use on the product and invoices includes this tax.")
+    price_include = fields.Boolean(
```

Comment:
Why keeping this field?
I guess it's because it's complicated and you are running out of time to merge

Reply:
Yes, the thing is, tests are full of taxes that explicitly set `price_include` (also on purchase taxes), so I need the `set` and the `search` too.